src/util/overlays.ts

```diff
@@ -194,27 +194,29 @@ export const overlayState = (
   };
 };
 
-const moveTranslate = (point: Point) =>
-  point ? `translate(${point.x.toFixed(0)}px, ${point.y.toFixed(0)}px)` : '';
-
 export const overlayTransform = ({
   anchor,
   position,
   offset
-}: OverlayParams) => {
-  const res = [];
+}: OverlayParams): { left: number, top: number, transform: string } => {
+  const cssPosition = {
```

Comment:
Itt felesleges objektumot használni, elég 2 változó (left, top)

Reply:
A harmadik (`transform`) paraméter állítja be az ablak anchor-függő igazítását.